source/server-discovery-and-monitoring/tests/unified/standalone-topology-lifecycle.json

```diff
@@ -0,0 +1,78 @@
+{
```

Comment:
Should this file be removed? I notice there's no yaml verison.

Reply:
Good catch. Deleted.